python/ray/tune/schedulers/bohb.py

```diff
@@ -0,0 +1,62 @@
+import ray
```

Comment:
can you move this file to `tune/suggest`?

and add the three `__future__` imports to the top?